cms/templates/js/certificate-editor.underscore

```diff
@@ -28,9 +28,10 @@
          <header>
            <label class="title-2"><%= gettext("Course Signatories") %></label>
          </header>
-        <div class="signatory-list"> </div>
+        <div class="signatory-edit-list"> </div>
         <span>
-           <a class="button action-add-signatory" href="javascript:void(0);"><%= gettext("Add Signatory (max 4)") %></a>
+           <a class="button action-add-signatory" href="javascript:void(0);"><%= gettext("Add Signatory") %></a>
```

Comment:
Appears something has happened to the limiter here, and I can now add a 5th signatory -- invalid element/class reference in the editor.js file?